SemDiff.Core/GitHub.cs

```diff
@@ -76,7 +77,8 @@ public Task UpdateLimitAsync()
         private async Task<string> HttpGetAsync(string url)
         {
             //Request, but retry once waiting 5 minutes
-            if(EtagNoChanges != null)
+            paginationURL = null;
```

Comment:
Maybe at this point it would be better to pass an object that contains an Etag and a pagination url, that way we could change them in the object we passed in instead of in the GitHub object. I only say that because we may run into concurrency problems this way.